# HG changeset patch
# User John Donoghue
# Date 1444685444 14400
#      Mon Oct 12 17:30:44 2015 -0400
# Node ID 93d96da9ff3e2a8d2db0ff0806d2bd26479e7c37
# Parent  d35201e5ce5d112889c407683732665c4f06826a
Update uiXXXXX documentation for autogenerated properties

* scripts/gui/uitoolbar.m: added ref to autogenerated properties

* scripts/gui/uicontextmenu.m: added ref to autogenerated properties

* scripts/gui/uicontrol.m: added ref to autogenerated properties

* scripts/gui/uipanel.m: added ref to autogenerated properties

* scripts/gui/uipushtool.m: added ref to autogenerated properties

* scripts/gui/uitoggletool.m: added ref to autogenerated properties

diff --git a/scripts/gui/uicontextmenu.m b/scripts/gui/uicontextmenu.m
--- a/scripts/gui/uicontextmenu.m
+++ b/scripts/gui/uicontextmenu.m
@@ -25,16 +25,18 @@
 ## If @var{h} is omitted then a uicontextmenu for the current figure is
 ## created. If no figure is available, a new figure is created first. 
 ##
 ## If @var{h} is given then a uicontextmenu relative to @var{h} is created. 
 ## 
 ## Any provided property value pairs will override the default values of the created 
 ## uicontextmenu object.
 ##
+## Uicontextmenu propertes are documented at @ref{Uicontextmenu Properties}.
+##
 ## Examples:
 ##
 ## @example
 ## @group
 ## % create figure and uicontextmenu
 ## f = figure;
 ## c = uicontextmenu (f);
 ##
diff --git a/scripts/gui/uicontrol.m b/scripts/gui/uicontrol.m
--- a/scripts/gui/uicontrol.m
+++ b/scripts/gui/uicontrol.m
@@ -28,16 +28,18 @@
 ## If @var{parent} is omitted then a uicontrol for the current figure is
 ## created. If no figure is available, a new figure is created first. 
 ##
 ## If @var{parent} is given then a uicontrol relative to @var{parent} is created. 
 ## 
 ## Any provided property value pairs will override the default values of the created 
 ## uicontrol object. 
 ##
+## Uicontrol propertes are documented at @ref{Uicontrol Properties}.
+##
 ## Control of the type of uicontrol created is through the use of the @var{style} property.
 ## If no style property is provided, a push button will be created.
 ##
 ## Valid styles for uicontrol are:
 ##
 ## @table @asis
 ## @item @qcode{"checkbox"}
 ## Create a checkbox control that allows user on/off selection.
diff --git a/scripts/gui/uipanel.m b/scripts/gui/uipanel.m
--- a/scripts/gui/uipanel.m
+++ b/scripts/gui/uipanel.m
@@ -27,16 +27,18 @@
 ## If @var{parent} is omitted then a uipanel for the current figure is
 ## created. If no figure is available, a new figure is created first. 
 ##
 ## If @var{parent} is given then a uipanel relative to @var{parent} is created. 
 ## 
 ## Any provided property value pairs will override the default values of the created 
 ## uipanel object.
 ##
+## Uipanel propertes are documented at @ref{Uipanel Properties}.
+##
 ## Examples:
 ##
 ## @example
 ## @group
 ## % create figure and panel on it
 ## f = figure;
 ## p = uipanel ("title", "Panel Title", "position", [.25 .25 .5 .5]);
 ##
diff --git a/scripts/gui/uipushtool.m b/scripts/gui/uipushtool.m
--- a/scripts/gui/uipushtool.m
+++ b/scripts/gui/uipushtool.m
@@ -29,16 +29,18 @@
 ## created. If no figure is available, a new figure is created first.  If a figure is
 ## available, but does not contain a uitoolbar, a uitoolbar will be created.
 ##
 ## If @var{parent} is given then a uipushtools is created on the @var{parent} uitoolbar. 
 ## 
 ## Any provided property value pairs will override the default values of the created 
 ## uipushtool object.
 ##
+## Uipushtool propertes are documented at @ref{Uipushtool Properties}.
+##
 ## Examples:
 ##
 ## @example
 ## @group
 ## % create figure without a default toolbar
 ## f = figure ("toolbar", "none");
 ## % create empty toolbar
 ## t = uitoolbar (f);
diff --git a/scripts/gui/uitoggletool.m b/scripts/gui/uitoggletool.m
--- a/scripts/gui/uitoggletool.m
+++ b/scripts/gui/uitoggletool.m
@@ -29,16 +29,18 @@
 ## created. If no figure is available, a new figure is created first.  If a figure is
 ## available, but does not contain a uitoolbar, a uitoolbar will be created.
 ##
 ## If @var{parent} is given then a uitoggletool is created on the @var{parent} uitoolbar. 
 ## 
 ## Any provided property value pairs will override the default values of the created 
 ## uitoggletool object.
 ##
+## Uitoggletool propertes are documented at @ref{Uitoggletool Properties}.
+##
 ## Examples:
 ##
 ## @example
 ## @group
 ## % create figure without a default toolbar
 ## f = figure ("toolbar", "none");
 ## % create empty toolbar
 ## t = uitoolbar (f);
diff --git a/scripts/gui/uitoolbar.m b/scripts/gui/uitoolbar.m
--- a/scripts/gui/uitoolbar.m
+++ b/scripts/gui/uitoolbar.m
@@ -25,16 +25,18 @@
 ## If @var{parent} is omitted then a uitoolbar for the current figure is
 ## created. If no figure is available, a new figure is created first. 
 ##
 ## If @var{parent} is given then a uitoolbar relative to @var{parent} is created. 
 ## 
 ## Any provided property value pairs will override the default values of the created 
 ## uitoolbar object. 
 ##
+## Uitoolbar propertes are documented at @ref{Uitoolbar Properties}.
+##
 ## Examples:
 ##
 ## @example
 ## @group
 ## % create figure without a default toolbar 
 ## f = figure ("toolbar", "none");
 ## % create empty toolbar
 ## t = uitoolbar (f);
